source/retryable-writes/retryable-writes.rst

```diff
@@ -212,24 +212,34 @@ Determining Retryable Errors
 
 When connected to a MongoDB instance that supports retryable writes (versions 3.6+),
 the driver MUST treat all errors with the RetryableWriteError label as retryable.
-This label might be added to an error in a variety of ways:
+This error label can be found in the top-level "errorLabels" field of the error.
+With a WriteConcernError response, the top level document or the
```

Comment:
Are `WriteConcernError` Ruby specific? You might need to loosen the language a bit here, maybe something like "In a server error response with a `writeConcernError` field..."

Reply:
I think you're right about it being Ruby specific. Thanks for flagging that.